.github/workflows/build.yaml

```diff
@@ -358,10 +358,51 @@ jobs:
           ./build-scripts/ubuntu-1604/build-3rd-parties.sh ./cache/3rd-party-dependencies
           mv ./build-scripts/ubuntu-1604/cache/* /tmp/third-party-dependencies
 
+  build-python-packages:
+    name: Build Python Packages
+    runs-on: ubuntu-20.04
+    needs: [workflow-setup, indy_plenum_tests, indy_plenum_module_tests, lint]
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+
+      - name: Set up Python 3.5
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.5
+
+      - name: Install required packages via pip
+        run: |
+          python3 -m pip install pytest-runner wheel pip==9.0.3
```

Comment:
As of #1548 the pip version does not need to be overridden

Reply:
Thanks for the contribution, @andrewwhitehead. I update the PR and removed the pinned pip version.